docs/lesson-plans/week-1.md

```diff
@@ -102,42 +95,541 @@ every week may not end up helping you as much as it would for our project, and y
+>on using IntelliJ for all of this, but feel free to try those out if you're interested.
+
+### Cloning the repositories with Git
+
+Now that you've gotten IntelliJ set up, we'll get to cloning the repositories on your computer.
+
+>Note: you may need a GitHub account to do this. If you encounter an error trying to clone it,
+>then then just sign up and and retry the steps after logging into GitHub in IntelliJ.
+
+Inside of IntelliJ, go to File > New... > Project From Version Control..., make sure the 
+`Version Control` box is set to `Git`, paste in the following links, and choose where you 
+want them to be downloaded to. 
+
+The links you'll want to use are 
+- Backend: https://github.com/Code-4-Community/jumpstart-backend/tree/starter-code
+- Frontend: __*GET STARTER CODE TAG DONE*__
+
+Once you've done that, you're ready to install other programs we'll need and build the projects.
+
+### Installing required apps
+
+There are a few other programs you'll need to install before we're done here. 
+
+For the frontend and backend, open up the projects and start reading the README.md files by 
+double clicking them in the project pane on the upper left. If you don't see it, you may have to
+expand the pane by clicking on the vertically spelled "project" tab, and then opening up
+the dropdowns indicating folders.
+
+To make sure the formatted text pops up, once you open the README.md files, click on the image-like 
+icon in the upper right to view the formatted markdown. You can also try viewing it side-by-side as
+plaintext and formatted markdown if you're interested to see what it looks like.
+
+Troubleshooting: 
+- For the frontend, don't forget to install your dependencies before starting the project, or you'll
+get errors. 
+- For the backend, make sure that you set the `JAVA_HOME` variable and that everything else required
+is installed.
+- For Git bash, the terminal needs to be manually set in every project. Don't forget to do that, or
+else you'll end up running commands that don't exist in Windows.
+
+## Terminal Basics
+
+The terminal is a really important tool, which you'll find extremely useful throughout your 
+developer career. Not only does it allow you to access settings and commands which you would otherwise
+not have access to, it sometimes is just easier to be able to do some things by typing out commands
+rather than pointing and clicking with a mouse. 
+
+Learning how to use the terminal well may take a while, but we'll get you started with a few commands
+we think every developer should know. We'll also show you how to build your projects if you 
+haven't tried doing that yet.
+
+>Note: we'll be showing you how to use unix-based bash commands, which means that these commands will 
+>work for Mac and Linux computers. If you have a Windows computer, either download the Linux subsystem
+>or follow the commands in the backend README.md to install Git bash and start using that in your 
+>IntelliJ projects.
+
+### Common Commands
+
```

Comment:
So just a thought, but maybe we should start by explaining that every terminal instance is working out of one directory, and that commands may only work from certain directories. This may not be clear to users who have never used the terminal before

Reply:
Added a quick explanation of that